### DISCUSSION TITLE
Get details based on either expected standard deviation or expected mean return.

### DISCUSSION THREAD

**Discussion by deshiyan1010** [dcajasn/Riskfolio-Lib#45] [Q&A] [answered]
In Efficient Frontier, I wanted to know how can I get details like:

When I give a fixed expected mean return/expected standard deviation, I get back the expected standard deviation/expected mean return and weights given to the portfolio.

**Comment by deshiyan1010** [comment, marked as answer]
Refer to this: [Discussion 11](https://github.com/dcajasn/Riskfolio-Lib/discussions/11)

**Reply by dcajasn** [reply]
Here is the answer
